wandb/integration/langchain/wandb_tracer.py

```diff
@@ -278,16 +237,14 @@ def load_default_session(self) -> "TracerSession":
         self._session = TracerSession(id=1)
         return self._session
 
-    def _add_child_run(
-        self,
-        parent_run: Union["ChainRun", "ToolRun"],
-        child_run: Union["LLMRun", "ChainRun", "ToolRun"],
-    ) -> None:
-        """Add child run to a chain run or tool run."""
-        try:
-            parent_run.child_runs.append(child_run)
-        except Exception:
-            # Silently ignore errors to not break user code
-            pass
-
     # End of required methods
+
+
+def autolog(run_args: Optional[WandbRunArgs] = None):
+    monkeypatch.clear_patches()
+    tracer = WandbTracer(run_args=run_args)
+    monkeypatch.ensure_patched(tracer=tracer)
+
+
+def disable_autolog():
+    monkeypatch.clear_patches()
```

Comment:
Perhaps we want to make it consistent with how we do autolog for OpenAI? See https://github.com/wandb/wandb/blob/main/tests/functional_tests/t0_main/openai/t2_openai_completion.py for an example.